cli/utilities/repos.py

```diff
@@ -0,0 +1,32 @@
+from cli.exceptions import ResourceNotFoundError
+from cli.utilities.packages import Package
+from cli.utilities.utils import os_major_version
+from utility.log import Log
+from utility.utils import get_cephci_config
+
+log = Log(__name__)
+
+
+def setup_local_repos(ceph):
+    # Get configuration details from `~/.cephci.yaml`
```

Comment:
Reading from `cephci.yaml` is going to put us in the same spot as `registry`. IMHO, we should avoid it.

This is required for the base image and it can come from the inventory file as we control what is getting installed.

Reply:
In case of `baremetal`, this would definitely be useful but it should be an input passed via `custom-config` to `install_prereq`